dev/com.ibm.websphere.appserver.features/visibility/public/passwordUtilities-1.1/com.ibm.websphere.appserver.passwordUtilities-1.1.feature

```diff
@@ -0,0 +1,23 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.passwordUtilities-1.1
+WLP-DisableAllFeatures-OnConflict: false
```

Comment:
It's a new feature.  You can remove this line and make it go to true by default.

Reply:
Made the change.